tests/system/CodeIgniterTest.php

```diff
@@ -792,9 +793,15 @@ public function testPageCacheWithCacheQueryString(
 
             // Cache the page output using default caching function and $cacheConfig
             // with value from the data provider
+            ob_start();
             $this->codeigniter->run();
-            // Cache the page using our own $cacheConfig confugration
-            $this->codeigniter->cachePage($cacheConfig);
+            $output = ob_get_clean();
+
+            $this->assertSame($string, $output);
+
+            if (count(ob_list_handlers()) > 1) {
```

Comment:
Why not `ob_get_level()`?

Reply:
Because I copied from the tearDown().

Changed.